### PR TITLE
Check if $color has a value before checking this value.

### PR DESCRIPTION
**Pull request by JoryHogeveen**
Fixes: #4919

### CODE REVIEW

File: classes/fields/color.php [hunk 1]
```diff
@@ -111,7 +111,7 @@ public function validate( $value, $name = null, $options = null, $fields = null,
 					// @todo Ask for a specific format in error message
 					$errors[] = __( 'Invalid value provided for this field.', 'pods' );
 				}
-			} elseif ( ! in_array( strlen( $color ), array( 3, 6 ), true ) ) {
+			} elseif ( $color && ! in_array( strlen( $color ), array( 3, 6 ), true ) ) {
```
Comment (pglewis) [inline classes/fields/color.php]: Not sure it'll make any difference in this context, but lets change it to `! empty( $color )` to match the logic before the breakage

Comment (JoryHogeveen) [reply classes/fields/color.php]: Won't matter since it's already declared but fine with me. Also enhances readability so that's a plus :)

**Comment by JoryHogeveen** [comment]
I think it would be good to add some unit tests to this PR. I'll get to that a.s.a.p. tomorrow unless anyone else wants to jump in.

**Comment by pglewis** [comment]
I'm all for adding tests but I'd rather create a separate issue and PR for them.

**Comment by sc0ttkclark** [comment]
@JoryHogeveen :taco: just ran into this in my own testing (without realizing i was running 2.7.3), you are a saint!

**Pull request merged**